Sources/Valkey/Connection/ValkeyConnection.swift

```diff
@@ -161,16 +176,48 @@ public final actor ValkeyConnection: ValkeyClientProtocol, Sendable {
 
     @inlinable
     func _execute<Command: ValkeyCommand>(command: Command) async throws -> RESPToken {
+        #if DistributedTracingSupport
+        let span = startSpan(Command.name, ofKind: .client)
```

Comment:
We should 100% inject a tracer into the ValkeyConnection: `let tracer: (any Tracer)?` We should never go through the global. We might want to get the global when we create the Connection.